Modules/_datetimemodule.c

```diff
@@ -1435,8 +1440,7 @@ new_timezone(PyObject *offset, PyObject *name)
         GET_TD_DAYS(offset) < -1 || GET_TD_DAYS(offset) >= 1) {
         PyErr_Format(PyExc_ValueError, "offset must be a timedelta"
                      " strictly between -timedelta(hours=24) and"
-                     " timedelta(hours=24),"
-                     " not %R.", offset);
```

Comment:
Instead of a blanket approach, I think each case should be considered on its own. For example for this specific line, the result is a _less informative_ error message.

Reply:
@erlend-aasland I would like to leave the general template of the error message
and I only now realized why it was not possible to pass the tests when I changed the error message in another place